go.mod

```diff
@@ -70,7 +70,7 @@ require (
 	golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2
 	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
 	gopkg.in/gormigrate.v1 v1.6.0
-	gopkg.in/guregu/null.v2 v2.1.2 // indirect
+	gopkg.in/guregu/null.v2 v2.1.2
 	gopkg.in/guregu/null.v3 v3.4.0
```

Comment:
Running 'go mod tidy' suggests this change is irrelevant.  Up for taking it out :)